auth/server/src/main/java/com/seagates3/response/generator/UserPolicyResponseGenerator.java

```diff
@@ -56,4 +61,20 @@ ServerResponse detachNonAttachedPolicy() {
     return formatResponse(HttpResponseStatus.NOT_FOUND, "NoSuchEntity",
                           errorMessage);
   }
+
+ public
+  ServerResponse generateAttachedUserPolicyListResponse(
+      List<Policy> policyList) {
+    ArrayList<LinkedHashMap<String, String>> policyMembers = new ArrayList<>();
+    LinkedHashMap responseElements;
+    for (Policy policy : policyList) {
+      responseElements = new LinkedHashMap();
+      responseElements.put("PolicyName", policy.getName());
```

Comment:
PolicyName, PolicyArn should be used form constants

Reply:
Will be taken care in a separate ticket EOS-26886.